src/client/containers/QuizPage/QuizPage.component.js

```diff
@@ -1,3 +1,43 @@
 import React from 'react';
+import './QuizPage.styles.css';
+import PageHeader from '../../components/PageHeaderComponent/PageHeader';
+import QuestionComponent from '../../components/Question/QuestionComponent';
+import QuizAnswers from '../../components/QuizAnswers/QuizAnswers';
+import SideMenu from '../../components/sideMenubar/sideMenu';
+import Buttons from '../../components/Buttons/Buttons';
+import ProgressBar from '../../components/ProgressBar.components/ProgressBar';
+import image1 from '../../assets/images/questionBackgrounds/question1background.png';
 
-export const QuizPage = () => <section>Quiz Page</section>;
+export const QuizPage = () => {
+  return (
+    <div className="main">
+      <div>
+        <PageHeader />
+      </div>
+      <div className="middle_page">
+        <div className="sidemenu_page">
+          <SideMenu highLigtItem={3} />
+        </div>
+        <div className="question_page">
+          <QuestionComponent
+            imageSrc={image1}
+            question="When visiting a website, what is that you are most interested in?"
+          />
+          <div className="quizans_page">
```

Comment:
Charmi, I didn`t find this class in your css file, so maybe it should be deleted.

Reply:
Thank you so much @sofiiadidovych for review my PR.. i will do.. :)